src/main/java/cz/tstrecha/timetracker/util/FilterUtils.java

```diff
@@ -0,0 +1,8 @@
+package cz.tstrecha.timetracker.util;
+
+public class FilterUtils {
+
+    public static String enrichLikeStatements(String input){
+        return "%" + input.toLowerCase() + "%";
```

Comment:
Naming `enrichLikeStatements` isn't the best, since it doesn't only enrich statement with "%", but also turns it to lowercase. I would move the .toLowercase somewhere else and make this method just adding "%".

Reply:
After rethinking - thats alright.